misc/docker-ci/check.mk

```diff
@@ -24,37 +25,38 @@ ossl1.1.0+fuzz:
 	docker run $(DOCKER_RUN_OPTS) $(CONTAINER_NAME) \
 		env CC=clang CXX=clang++ \
 		make -f $(SRC_DIR).ro/misc/docker-ci/check.mk _check \
-		CMAKE_ARGS='-DOPENSSL_ROOT_DIR=/opt/openssl-1.1.0 -DBUILD_FUZZER=ON' \
+		CMAKE_ARGS="$(CMAKE_ARGS) -DOPENSSL_ROOT_DIR=/opt/openssl-1.1.0 -DBUILD_FUZZER=ON" \
```

Comment:
Do we need this change and the changes below? 

We are setting the default for each invocation of `check.mk` right above at line 5.